syntax = "proto3";

package website_builder;

message FeatureItem {
  string title_i18n_key = 1;
  string desc_i18n_key = 2;
  // Optional: Add if icons or specific images per feature are needed
  // string icon_class = 3;
  // string img_src = 4;
  // string img_alt_i18n_key = 5;
}
